cmd/daemon/main.go

```diff
@@ -92,35 +94,41 @@ var (
 		Flags: []cli.Flag{
 			&cli.UintFlag{
 				Name:  flagRPCPort,
-				Usage: "port for the daemon RPC server to run on; default 5001",
+				Usage: "Port for the daemon RPC server to run on",
+				Value: defaultRPCPort,
```

Comment:
There were a bunch flags reporting incorrect values and multiple defaults. Using the `Value` field (default value), will make the code easier to maintain.

Old version:
```
--rpc-port value     port for the daemon RPC server to run on; default 5001 (default: 0)
```
New Version:
```
--rpc-port value     Port for the daemon RPC server to run on (default: 5005)
```